crates/stackable-operator/src/utils/kubelet.rs

```diff
@@ -0,0 +1,69 @@
+use http;
+use k8s_openapi::api::core::v1::Node;
+use kube::{
+    Api,
+    api::{ListParams, ResourceExt},
+    client::Client,
+};
+use serde::Deserialize;
+use snafu::{OptionExt, ResultExt, Snafu};
+
+use crate::commons::networking::DomainName;
+
+#[derive(Debug, Snafu)]
+pub enum Error {
+    #[snafu(display("failed to list nodes"))]
+    ListNodes { source: kube::Error },
+    #[snafu(display("failed to build proxy/configz request"))]
+    ConfigzRequest { source: http::Error },
+
+    #[snafu(display("failed to fetch kubelet config from node {node}"))]
```

Comment:
This way it's quoted and special chars are correctly escaped
```suggestion
    #[snafu(display("failed to fetch kubelet config from node {node:?}"))]
```